src/web/components/StickyBottomBanner/BrazeBanner.test.ts

```diff
@@ -93,37 +93,9 @@ describe('hasRequiredConsents', () => {
 });
 
 describe('canShowPreChecks', () => {
-	describe('when the switch is off', () => {
```

Comment:
While the refactor looks correct, it means the logic for checking the switch is now all intertwined with the side effects of the integration. It follows that we no longer have test coverage of these cases. Are we happy with that?

Reply:
Also discussed offline, but yep I think it's acceptable (the switch/apiKey check is super simple).